ci-operator/step-registry/baremetalds/two-node/fencing/post-install/node-degredation/baremetalds-two-node-fencing-post-install-node-degredation-commands.sh

```diff
@@ -0,0 +1,224 @@
+#!/bin/bash
+set -o nounset -o errexit -o pipefail
+
+log(){ echo "[$(date +'%F %T%z')] $*"; }
+echo "[INFO] degraded two-node fencing pre step starting..."
+
+# config / artifacts
+ART_BASE="${ARTIFACT_DIR:-/tmp/artifacts}/degraded-two-node"
```

Comment:
To avoid any accidents, assume the `ARTIFACT_DIR` is present since it's part of supplied env vars. https://docs.ci.openshift.org/docs/architecture/step-registry/#available-environment-variables